README.md

```diff
@@ -21,6 +21,10 @@ Add the plugin to your npm-project:
 $ npm install semantic-release-slack-bot -D
 ```
 
+## Requirements
+
+As per the new release of semantic-release library version 20, the library is requiring the use of Node v18, which we need to comply as well to use keep the repo updated with it's latest feature. More information can be found [here](https://github.com/semantic-release/semantic-release/releases/tag/v20.0.0).
+
```

Comment:
I don't think this comment is correct, since this library currently also works with semantic release < 20 and node 14.x ....

Reply:
Check here: https://github.com/semantic-release/semantic-release/commit/8a0d8be51f4bf21e10bc0cd41ffb5ce475112442

Reply:
Even though both semantic-release and semantic-release-slack-bot works with node 14.x for now, it could change in the future. And I think it makes sense to not overpromise. If node 14.x is required you shouldn't upgrade this library.

And semantic-release have stopped testing their fixes on node v14 so if they introduce some code that breaks in node v14 it might go undetected.

Do you still think that this isn't a breaking change? Maybe I haven't thought about all the consequences.

Reply:
> Do you still think that this isn't a breaking change? Maybe I haven't thought about all the consequences.

What I mean is, that people that are still using (or need to use) semantic-release 19.x with node 14.x. still should be able to use semantic-release-slack-bot. there has nothing changed in semantic-release-slack-bot that would make that impossible.

> And semantic-release have stopped testing their fixes on node v14 so if they introduce some code that breaks in node v14 it might go undetected.

yes. but this is not the concern of semantic-release-slack-bot. only when they make API changes (which will they do with a new major release) there will be a problem.

Reply:
I think you leave good concerns that users might want to use the latest version of `semantic-release-slack-bot` but not the latest version of `semantic-release`. I don't think we should prohibit it but at the same time I don't think we need to ensure that this version is compatible. So you are correct that we might not need to write anything about it in the readme.